tutorials/scripting/gdscript/warnings/unreachable_code.rst

```diff
@@ -0,0 +1,62 @@
+UNREACHABLE_CODE
+====================
+
+The warning message is:
+
+.. code-block:: none
+
+    Unreachable code (statement after return) in function "_ready()".
+
+The default warning level for this warning is **Warn**.
+To modify it, see :ref:`ProjectSettings.debug/gdscript/warnings/unreachable_code<class_ProjectSettings_property_debug/gdscript/warnings/unreachable_code>`.
+
+When this warning occurs
+------------------------
+
+This warning may appear if there is code in a function that can never be reached because all possible code paths before it would hit a ``return`` statement and leave the function:
```

Comment:
A bit ambiguous because both the next "_that_" and "_it_" may refer to "_in a function_" as the subject.
```suggestion
This warning may appear if there is code that can never be reached because all possible code paths before it would hit a ``return`` statement and leave a function:
```
or something else to mitigate the ambiguity, I am not sure